tests/view/rooms/room/app.js

```diff
@@ -114,4 +116,18 @@ describe('tests/view/rooms/room/app.js, /rooms/{key} =>', () => {
     const redirectUrl = app.getRedirectUrl(href, roomId);
     expect(redirectUrl).to.equal(`/rooms/${roomId}${paramString}`);
   });
+
+  it('ok, "not found modal" element is in html of room from pug file', () => {
+    document.body.innerHTML = ui;
+    expect(document.getElementById('room_not_found_modal').innerHTML).to.not.equal(null);
+  });
+
+  it('ok, "not found modal" is hidden by default', () => {
```

Comment:
since you are checking the default behavior (else condition from you fn), would be nice checking:
* goToCreateRoomsButton.onclick &  gotToRoomListButton.onclick has valid
* notFoundText.innerText = has not found message

This will the else